### PR TITLE
add back in staticdate hack

### PR DESCRIPTION
**Pull request by wpride**
Add back in static date hack that we need, removed in 

https://github.com/dimagi/touchforms/pull/208/files

cross https://github.com/dimagi/commcare-hq/pull/11641

### CODE REVIEW

File: touchforms/backend/handlers/static.py [hunk 1]
```diff
@@ -33,3 +35,38 @@ def eval(self, args, ec):
...
+                parsed_value = SimpleDateFormat('yyyy-MM-dd').parse(value)
+            else:
+                # assume datetime format
+                # remove microseconds if necessary
+                if len(value) == 27:
+                    value = '%sZ' % value[:19]
+                sdf = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'")
+                sdf.setTimeZone(TimeZone.getTimeZone("UTC"))
+                parsed_value = sdf.parse(value)
+            self._value = parsed_value
```
Comment (dimagimon) [inline touchforms/backend/handlers/static.py]: W292 no newline at end of file

File: touchforms/backend/handlers/static.py [hunk 2]
```diff
@@ -33,3 +35,39 @@ def eval(self, args, ec):
...
+
```
Comment (dimagimon) [inline touchforms/backend/handlers/static.py]: W293 blank line contains whitespace
W391 blank line at end of file

**Comment by wpride** [comment]
commit comment should read "dimagimon I will kill you someday" not dimagi

**Comment by czue** [comment]
:+1: on :white_check_mark:

**Pull request merged**